policykit/policykit/settings.py

```diff
@@ -24,6 +24,8 @@
     ALLOWED_HOSTS=(list, []),
     SERVER_URL=(str, "http://127.0.0.1:8000"),
     LOG_FILE=(str, "debug.log"),
+    DATABASE_URL=(str, "sqlite://" + os.path.join(BASE_DIR, 'db.sqlite3')),
```

Comment:
I just tried it locally without docker and realized that we need one more `/` to make it an absolute path.

```suggestion
    DATABASE_URL=(str, "sqlite:///" + os.path.join(BASE_DIR, 'db.sqlite3')),
```